IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.h

```diff
@@ -55,6 +55,7 @@ MSIDWebviewUIController <MSIDWebviewInteracting, WKNavigationDelegate>
                         decisionHandler:(void (^)(WKNavigationActionPolicy))decisionHandler;
 
 @property (atomic, readonly) NSURL *startURL;
+@property (nonatomic, readonly) NSDictionary<NSString *, NSString *> *customHeaders;
```

Comment:
Will it always be as String and String type in the dict? Would it be expanded to other types in future?

Reply:
customHeaders is already part of the webviewController.m as private property. I am just exposing the readyonly version of it.